src-tauri/src/database/settings.rs

```diff
@@ -0,0 +1,27 @@
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize, Clone, Debug)]
+#[serde(rename_all = "camelCase")]
```

Comment:
Why?

This is synced to disk, we don't need the camelCase. camelCase is only for server responses.